examples/scala-sbt/src/main/scala/spark/bigtable/example/WordCount.scala

```diff
@@ -56,8 +56,12 @@ object WordCount extends App {
     )
     .drop("frequency_double")
 
+  // "bigtable" is not working - throwing Data Source not found
+  // when full path like below is provided, it works!
+  val bigtableFormat = "com.google.cloud.spark.bigtable.BigtableDefaultSource"
```

Comment:
Does this work without this PR's changes? If so this would be a breaking change and we should try to understand why it is failing

Reply:
Resolved. Now it is working with "bigtable"